web/eforbidden.jsp

```diff
@@ -17,14 +17,15 @@ information: Portions Copyright [yyyy] [name of copyright owner]
 CDDL HEADER END
 
 Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
---%><%@page  session="false" isErrorPage="true" import="
```

Comment:
I removed the definition that the eforbidden is a handler for 403 errors; in order not to confuse anybody I removed the declaration from this place as well.

This parameter in jsp means that there will be an `exception` object available for this jsp which caused the error which led to this jsp.